backend/models/outfitModel.js

```diff
@@ -14,11 +14,10 @@ const OutfitSchema = new mongoose.Schema({
       type: String
     }
   ],
-  worn: [
-    {
-      type: Date
-    }
-  ],
+  worn: {
```

Comment:
I originally planned this as an array of dates, since you might wear an outfit more than once and some people might want to track that. Keeping it as an array would also allow us to do implement stretch goals like "frequently worn outfits" or planning outfits to be worn on specific dates. And if we only want to display one date on the card, we could say something like "Last worn on ${date}"

Is there a compelling reason to use a single date here instead?

Reply:
i had changed it mainly since i wasnt sure why it was in an array format, if u want to track all the dates that is fine, ill just need to change the method used to interact with the date updates